shared/people/index.d.ts

```diff
@@ -8,6 +8,7 @@ export type Props = {
   followSuggestions: Array<Types.FollowSuggestion>
   getData: (markViewed?: boolean) => void
   onClickUser: (username: string) => void
+  onOpenAccountSwitcher?: (() => void) | null
```

Comment:
i'm fixing up a bunch of this in the ts strict branch. this basically allows the value to be a function or null or undefined. we can collapse it by removing the null . if you need it to not be a function just set it as undefined